Controllers/EmployeeController.cs

```diff
@@ -16,12 +16,105 @@ namespace TodoApi.Controllers
     public class EmployeeController : ControllerBase
```

Comment:
Avoid blocking async calls by using `await` instead of `.Result` to prevent potential deadlocks and improve performance.